app/frontend/src/uploadDocuments.js

```diff
@@ -0,0 +1,8 @@
+// This triggers a file input when the button is clicked
+
+document.addEventListener('DOMContentLoaded', function() {
+  document.getElementById('file-upload').addEventListener("click", function(e) {
+    e.preventDefault();
+    document.getElementById('upload').click();
+  });
+})
```

Comment:
Please fix the editor so that it automatically adds a new line at the end of the file so that we do not have end of line noise during reviews. GitHub also suggests it here